io500-info-editor.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+import sys
+import json
+
+# This tool allows to view and edit the options in an existing schema file.
+# In order to get started download the two files:
+# wget https://www.vi4io.org/lib/plugins/newcdcl/scripts/schema-io500.json
+# wget https://www.vi4io.org/lib/plugins/newcdcl/scripts/site-io500.json
+#
+# You can then copy the site-io500.json template file and modify it using this tool
+
+if len(sys.argv) < 3:
+  print("synopsis: %s <FILE> {<TOKEN[=VALUE]>}" % sys.argv[0])
+  print("examples:")
+  print("printing current: %s site.json Site.institution" % sys.argv[0])
+  print("Token examples:")
+  print("changing: Site.StorageSystem.Lustre.OSS.count=5")
```

Comment:
Should each of these example lines also have `"%s site.json ..." % sys.argv[0]`?  Otherwise, they aren't really clear in terms of _how_ to change these values, just that they could be changed.

Reply:
Also, it would be very useful to have a method of dumping all of the available options in the template, so that the user has some way to know what fields are available to be set.